monitoring/ceph-mixin/Makefile

```diff
@@ -0,0 +1,27 @@
+all: fmt generate lint test
+
+fmt:
+	./lint-jsonnet.sh -i
+
+generate: dashboards_out  prometheus_alerts.yaml
+
+vendor:
+	jb install
+
+dashboards_out: vendor
+	jsonnet -J vendor -m $@ dashboards.jsonnet
```

Comment:
This same command (as well as the alerts one) is repeated in 3 different places: in this Makefile, in a shell script, and in a toxfile... I'd rather have a single source of truth for generating this. Is it viable?

Reply:
If you want to allow users customize the build, tox also allows you to pass [`{posargs}`](https://tox.wiki/en/latest/example/general.html) (not sure if that was the issue you mentioned here).